README.md

```diff
@@ -141,6 +141,7 @@ A curated list of awesome things related to [docsify](https://docsify.js.org)
 - [JS-Ninja](https://abpanic.github.io/Mastering-JavaScript/#/) - JavaScript course to start as begineer and upskill to clear staff/principal level interviews.
 - [Sakura Docs](https://sakura-docs.arkinsolomon.net) - A programming language designed for package installers.
 - [Ransomware.live](https://www.ransomware.live) - A monitoring ransomware's victims in near real-time. 
+- [hope](https://cmai.xin/#/) - A blog documenting.
```

Comment:
A blog documenting what? This is an incomplete sentence. May as well have no description.

Reply:
I checked the content and maybe `A blog documenting my growth process.` would be more appropriate.

Reply:
Thanks